move/gas_service/Move.lock

```diff
@@ -21,6 +21,6 @@ dependencies = [
 ]
 
 [move.toolchain-version]
-compiler-version = "1.25.1"
+compiler-version = "1.26.1"
```

Comment:
version should match `test.yaml` gh action. It should generally only be changed in a separate PR for clarity